episodes/08-timing.md

```diff
@@ -112,5 +105,8 @@ Temperature at iteration 7000: 24.9819
 The simulation took 20.1621 seconds
 Final temperature at the desired position after 7750 iterations is: 24.9671
 The greatest difference in temperatures between the last two iterations was: 0.00199985
-~~~
-{: .output}
+```
+
+::::::::::::::::::::::::::::::::::::: keypoints
+- "Use UNIX `time` command or instrument your Chapel code to measure performance."
```

Comment:
`time` is useful for casual inspection. To quantitatively analyze performance, you must instrument.

Reply:
I rewrote the key point about measuring code performance. Resolved through a commit.